### ISSUE TITLE
[Improvement] Remember the client plugin setting in server side

### ISSUE THREAD

**Issue by f0re1gnKey** [msgbyte/tailchat#195]
I have many device using tailchat, and i change new device frequently. 

When i switch to another one, i must reconfigure plugin setting once again, extremely annoying.

**Comment by moonrailgun** [comment]
yes, because of plugin setting is device base rather than account basic.

Just like vscode, it took several years of development before has setting synchronization plugin.

We can develop a plugin sync plugin, i will add it into backlog

**Comment by f0re1gnKey** [comment]
> yes, because of plugin setting is device base rather than account basic.
> 
> Just like vscode, it took several years of development before has setting synchronization plugin.
> 
> We can develop a plugin sync plugin, i will add it into backlog

Anyway, i suggest adding group default plugins feature.(exclude 3rd because they are insecure)

Each group has their special usage, some group need special plugins to be installed. 
I must repeat the plugin install instructions once and once again when a new member join in the group, it is annoying and inefficient + bad user experience.

It will be better when the new member join in the group, installing the default plugins automatically.
